build-executor/src/main/java/org/jboss/pnc/executor/DefaultBuildExecutor.java

```diff
@@ -446,6 +447,8 @@ private void destroyEnvironment(BuildExecutionSession buildExecutionSession) {
             } else {
                 userLog.warn("Unable to destroy environment. Most likely due to cancelled operation.");
             }
+        } catch (OpenShiftException e) {
+            log.error("OpenShift connection ERROR.", e);
```

Comment:
If the case is that the environment is not destroyed that have to be written in the log.